### PR TITLE
Search SDK: Fixing Nuspec so the package installs in Windows Phone 8.1 a...

### PR DESCRIPTION
**Pull request by bitsforthought**
...pps (non-Silverlight)

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@brjohnstmsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (brjohnst). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/Search/Microsoft.Azure.Search.nuspec [hunk 1]
```diff
@@ -30,9 +30,9 @@
   </metadata>
   <files>
     <file src="Search\**\*.cs" target="src" />
-    <file src="..\binaries\portable\Microsoft.Azure.Search.dll" target="lib\portable-net45+sl50+wp80+win" />
-    <file src="..\binaries\portable\Microsoft.Azure.Search.pdb" target="lib\portable-net45+sl50+wp80+win" />
-    <file src="..\binaries\portable\Microsoft.Azure.Search.xml" target="lib\portable-net45+sl50+wp80+win" />
+    <file src="..\binaries\portable\Microsoft.Azure.Search.dll" target="lib\portable-net45+wp8+wpa81+win" />
```
Comment (yugangw-msft) [inline src/Search/Microsoft.Azure.Search.nuspec]: this is in line now with rest libraries, but it is not the most accurate. Once my PR to get all library built for 4.5 merged, I will refresh all

**Comment by yugangw-msft** [comment]
lgtm

File: src/Search/Microsoft.Azure.Search.nuspec (hunk 1 above)
Comment (brjohnstmsft) [inline src/Search/Microsoft.Azure.Search.nuspec]: Thanks!

**Pull request merged**